packages/faceted-search/src/components/AddFacetPopover/AddFacetPopver.component.test.js

```diff
@@ -1,7 +1,6 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
-import { mount } from 'enzyme';
 import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
```

Comment:
RTL ?

Reply:
Already quite a big PR, wanted to separate concerns here by removing this from the PR same as the badge part